spec/requests/graphql/mutations/update_user_spec.rb

```diff
@@ -30,6 +31,9 @@ def query(user_id)
                     name
                     email
                     birthdate
+                    eras {
+                      id
+                    }
```

Comment:
Since I requested all era fields in the get_single_user spec, I thought just requesting id here was fine but let me know if you think I should request all of the fields here too!

Reply:
I think ID is fine!

Reply:
2nd Dan!